lib/reporting/reports/suppliers/helpers/line_items_access_helper.rb

```diff
@@ -5,31 +5,30 @@ module Reports
     module Suppliers
       module Helpers
         module LineItemsAccessHelper
-          def variant(line_items)
-            line_items.first.variant
+          def variant(line_item)
+            line_item.variant
           end
```

Comment:
Now most of these helpers are no longer required and can be removed ;)

Reply:
Not as of now I think they are used in multiple place :)